modules/ROOT/pages/mtf-metadata-testing.adoc

```diff
@@ -0,0 +1,363 @@
+= Metadata 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Test the metadata defined for your module.
+
+== Before You Begin
+
+* Know how to define metadata in your module. For more information, refer to xref:metadata.adoc[].
```

Comment:
I know that "Understand" is a 25-cent word and "Know" is a 5-cent word (as my previous editor used to say), but I wonder if it's more precise in this case.